calabash.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,13 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		89043ABF1C289A5D0067E22F /* LPConstants.h in Headers */ = {isa = PBXBuildFile; fileRef = 89043ABD1C289A5D0067E22F /* LPConstants.h */; };
```

Comment:
Can be resolved with: Xcode: remove LPConstants.h from Project Headers #313

This is a blocking comment.